### PR TITLE
Fix OpenAPIV3Schema Validation Issue

### PR DESCRIPTION
**Pull request by zzxwill**
Temporarily corrects spec.validation.openAPIV3Schema issue, and it would be removed
after this issue was fixed https://github.com/oam-dev/kubevela/issues/284.

### CODE REVIEW

**Comment by zzxwill** [comment]
Hit issue https://github.com/oam-dev/kubevela/issues/301. Need more time to fix this issue first.

**Review by wonderflow — APPROVED** [review]
LGTM

File: hack/chart/generate.go [hunk 1]
```diff
@@ -1,20 +1,112 @@
...
+				if err = ioutil.WriteFile(targetPath, data, os.ModePerm); err != nil {
+					fmt.Fprintln(os.Stderr, "failed to read file:", err)
+					return err
+				}
+			}
+		}
+		return nil
+	})
+	if err != nil {
+		return chartPath
```
Comment (wonderflow) [inline hack/chart/generate.go:109]: if there's an error, we should return it, or the orignal chart will be removed

Comment (zzxwill) [reply hack/chart/generate.go:109]: Every time hits an issue, it returns the original `chartPath`. This won't do anything bad and can the build can continue. I think you are right, an error should throw out and break the build process.

**Pull request merged**